Polygon_mesh_processing/test/Polygon_mesh_processing/test_pmp_clip.cpp

```diff
@@ -283,6 +284,46 @@ void test()
   assert(is_valid_polygon_mesh(tm1));
   CGAL::clear(tm1);
   CGAL::clear(tm2);
+
+  // non-manifold border vertices
+  std::stringstream ss;
+  ss << "OFF\n 5 2 0\n 0 0 0\n2 0 0\n4 0 0\n4 1 0\n0 1 0\n3 0 1 4\n3 1 2 3\n";
```

Comment:
Note that C++11 has raw string literals, that allows multi-lines strings.
https://en.cppreference.com/w/cpp/language/string_literal#Example

Reply:
Milestone is 4.14.1

Reply:
But thanks for the ref :)